lib/team.rb

```diff
@@ -0,0 +1,32 @@
+require "pry"
+
+class Team
+  attr_reader :country, :players, :eliminated
+  def initialize(country)
+    @country = country
+    @players = []
+    @eliminated = false
+  end
+
+  def eliminated?
+    @eliminated
+  end
+
+  def eliminated
+    @eliminated = true
+  end
+
+  def add_player(player)
+    @players << player
+  end
+
+  def players_by_position(position)
```

Comment:
The logic on this looks really great!  A refactor you might consider is using `find_all`.  I've included an example below:

```ruby
def players_by_position(position)
  @players.find_by do |player|
    player.position == position
  end
end
```